packages/core/src/storybook/stand-alone-documentaion/welcome/other-contributors-list/other-contributors-list.tsx

```diff
@@ -60,6 +60,7 @@ export const OtherContributorsList = () => {
       </Paragraph>
       <GithubContributorsList
         organizationName="mondaycom"
+        // TODO: need to support more than 1 package
```

Comment:
wdym?

Reply:
Actually it's per repo, not npm package, changing to "vibe"